### PR TITLE
[WIP] Address verification from csv

### PR DESCRIPTION
**Pull request by peterdolan**
What: Simple refactoring of letters example, pulling code style from a previous refactoring done in letters csv
Why: Cleaning up examples to make python more 'pythony'

### CODE REVIEW

File: examples/verify_addresses_from_csv/verify.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+except Exception, e:
+    print('Error: ' + str(e))
+    print('Failed to create output directory. Aborting all sends.')
+    sys.exit(1)
+
+# output csv names
+success_filename = os.path.join(output_dir, 'success.csv')
+errors_filename = os.path.join(output_dir, 'errors.csv')
+
+try:
```
Comment (sjl2) [inline examples/verify_addresses_from_csv/verify.py]: Thoughts on removing this? It leads to some major indenting, but it probably is better...

File: examples/verify_addresses_from_csv/verify.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    with open(input_filename, 'r') as input, \
+      open(success_filename, 'w') as success, \
+      open(errors_filename, 'w') as errors:
+
+    # print mode to screen
```
Comment (sjl2) [inline examples/verify_addresses_from_csv/verify.py]: This next part probably isn't necessary for address verification.

File: examples/verify_addresses_from_csv/verify.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+        mode = lob.api_key.split('_')[0]
+        print('Sending letters in ' + mode.upper() + ' mode.')
+
+        input_csv = csv.DictReader(input)
+        errors_csv_fields += input_csv.fieldnames
+
+        success_csv = csv.DictWriter(success, fieldnames=success_csv_fields)
```
Comment (sjl2) [inline examples/verify_addresses_from_csv/verify.py]: Did this output style work well for the verified addresses? Do bad ones go to errors?

Comment (sjl2) [inline examples/verify_addresses_from_csv/verify.py]: Also, do you know how to write the fields to the first row of the csv?

File: examples/verify_addresses_from_csv/verify.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../../..'))
+
+# TODO: Provide your API key, keep this secure
+lob.api_key = '<insert your api key here>'
```
Comment (sjl2) [inline examples/verify_addresses_from_csv/verify.py]: I'm not sure how we had it on the other ones, but we should keep it consistent. I think it's `your key here` on the others. I like the idea of `your api key here`.

Comment (sjl2) [inline examples/verify_addresses_from_csv/verify.py]: I agree with your comments on mine. Let's move this to `''`

**Comment by sjl2** [comment]
Oh wait, @peterdolan can you update the [README](https://github.com/lob/lob-python/blob/address_verification_from_csv/examples/README.md) first?

**Comment by mgartner** [comment]
Marking as WIP until this is ready for review.

**Comment by peterdolan** [comment]
@sjl2 I don't understand what you want me to update -- README appears to have verify addresses from csv?

**Comment by peterdolan** [comment]
@sjl2 FIXED lgtm me boi

**Comment by sjl2** [comment]
Make sure you rebate! @peterdolan

**Comment by sjl2** [comment]
@peterdolan Looks like you're going to have to rebase off the removal of jobs. I guess it was using the API of an account we are no longer using.

**Pull request closed without merging**